akka-http/src/main/scala/akka/http/scaladsl/server/PathMatcher.scala

```diff
@@ -48,7 +48,7 @@ abstract class PathMatcher[L](implicit val ev: Tuple[L]) extends (Path => PathMa
   }
 
   /** Operator alternative to [[PathMatchers.not]] */
-  def unary_!(): PathMatcher0 = PathMatchers.not(self)
+  def unary_! : PathMatcher0 = PathMatchers.not(self)
```

Comment:
ha ;)

Reply:
Btw. needs that extra space to compile, I guess that might be a reason the parentheses were added in the first place.